src/components/activity/ActivityDecomposition.svelte.test.ts

```diff
@@ -175,6 +175,8 @@ const activities: Activity[] = [
 
 describe('Activity Decomposition component', () => {
   beforeAll(() => {
+    // See: https://github.com/sveltejs/kit/issues/6259
```

Comment:
Curious, did this just start failing as a result of this PR or was it separate? Don't mind it being here of course.

Reply:
This was from this PR actually. I moved the `activityMetadataDefinitions` store to the `activities.ts` file, and we are importing from `activities.ts` in `ActivityDecomposition.svelte.test.ts`. Since `activityMetadataDefinitions` is now a `gqlSubscribable`, that `subscribeable.ts` file imports `import { browser } from '$app/environment';`, which was causing this error. Very weird one but at least there is a workaround for now. I subscribed to that Svelte Kit issue so we can remove the workaround when they fix it.